### ISSUE TITLE
logo shows diffs if files renamed, even if unchanged.

### ISSUE THREAD

**Issue by bobtfish** [okta/terraform-provider-okta#1036]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.13.7

### Affected Resource(s)

- okta_app_basic_auth
- okta_app_bookmark
- okta_app_oauth
- okta_app_saml

### Terraform Configuration Files

```hcl
resource "okta_app_saml" "example" {
  ...
  logo = "example.png"
  ...
}
```

### Expected Behavior

Resource does not show diffs if logo file is renamed without any other changes

### Actual Behavior

Resource shows diff/change if logo is renamed

### Steps to Reproduce

1. Create an application with Terraform, including a logo file.
2. `terraform apply`
3. `mv example_logo.png new_example_logo.png`
4. Change code to reference the new logo file
5. `terraform plan`

Expect to see no changes (as the logo itself has not changed, just the location on the local filesystem), do see changes.

- #0000

**Comment by monde** [comment]
Thanks for the PR #1037 @bobtfish , I'll take a look.

**Comment by monde** [comment]
Addressed in #1039

**Comment by bobtfish** [comment]
Thanks for the super quick response @monde - much appreciated.

If we can get a point release in a timely fashion too that would be amazing!
